### PR TITLE
Update manifiesto

### PR DESCRIPTION
**Pull request by Titogelo**
See how the new items relate to the old ones:
- 1: to the old 1
- 2: to the old 10, 3, 2. Socio-emotional skills and 4C communication, critical thinking, collaboration and creativitiy are by-products of STEAM.
- 3: to the old 6, 7, 
- 4: to the old 11, and adds the gender discussion which was missing.
- 5: to the old 4.
- 6: I think we need this but it might go against the values `people and relationships over goals and objectives`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
+- People and relationships over goals and objectives<br>
+- Think beyond the limits established over initiatives and conformist thoughts<br>
 </p>
 
 That is, while there is value in the items on the right, we value the items on the left more.
 
 We, as implementers, follow these principles:
 
-1. The highest priority is to make the represented region visible on the international STEM landscapes.
-1. All initiatives are driven by transparency, honesty, and respect people.
```
Comment (marcosflobo) [inline README.md:21]: - [ ] I like this one, I see it really important in the current era, with so much noise, and people fighting each other. I would propose keeping it.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-1. Promote the sustainable development, within the region we represent, through STEM.
-1. Define a clear range of alignment options which must be followed by the participants.
-1. The promoted initiatives are meant to have the highest impact possible.
-1. Exchange among people and cultures must be encouraged.
```
Comment (marcosflobo) [inline README.md:25]: - [ ] I like this one as well. I see important the point about exchanging with other parts of the world

File: README.md [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
-1. Work with a committed set of people.
-1. Celebrate the wins, even the small ones, and hold on the bad moments.
-1. Hope for the best, be ready for the worst, and enjoy the change.
-1. Continuous improvement should be advocated at any time based on feedback.
```
Comment (marcosflobo) [inline README.md:29]: - [ ] This one as well I like. Why? I think one of the mistakes that many organizations have is the lack of retrospectives and feedback, to be sure their strategy is continuously adapted (an organization is a living being!)

**Review by marcosflobo — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,33 +1,29 @@
...
-1. Always plan strategy thinking on the future of the region and its people, not only the present day.
+1. Boost the region's visibility and competitiveness on international STEAM landscapes.
+2. Promote socio-emotional skills and 4 c's of learning 21st century essential for the development of a region.
```
Comment (marcosflobo) [inline README.md:21]:
```suggestion
2. Promote socio-emotional skills and Critical Thinking, Communication, Collaboration, and Creativity of learning 21st century essential for the development of a region.
```

**Review by marcosflobo — APPROVED** [review]

**Review by cloudnativeasturias — APPROVED** [review]

**Pull request merged**